<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>208</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="name_input">
         <property name="placeholderText">
          <string>Nama</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nim_input">
         <property name="placeholderText">
          <string>NIM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tl_input">
         <property name="placeholderText">
          <string>Tanggal lahir (yyyy-mm-dd)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="search_input">
       <property name="placeholderText">
        <string>Search by name...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Simpan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="export_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ekspor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table">
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="hapus_button">
       <property name="text">
        <string>Hapus Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Gagah Putra Lesmana - F1D021037</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSimpan"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCari"/>
    <addaction name="actionHapus"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSimpan">
   <property name="text">
    <string>Simpan</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Ekspor ke CSV</string>
   </property>
  </action>
  <action name="actionCari">
   <property name="text">
    <string>Cari Judul</string>
   </property>
  </action>
  <action name="actionHapus">
   <property name="text">
    <string>Hapus Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
